cogs/games.py

```diff
@@ -20,18 +20,203 @@
+                        await hg_msg.edit(embed=txt_embed)
+                        if len(not_guessed) == 0:
+                            await ctx.send(
+                                f"congratulations {curr_msg.author}, you solved the hangman"
+                            )
+                    elif curr_guess in word:    # curr_guess not in not_guessed (elif) and in word => curr_guess is correct but already made
+                        player_msg_list.append(
+                            f"{curr_msg.author}, '{curr_guess}' was already guessed"
+                        )
+                        if len(player_msg_list) > 3:
+                            player_msg_list = player_msg_list[-3:]
+                        txt_embed.set_field_at(
+                            0,
+                            name=f"hangman (category: {cat_name})",
+                            value=
+                            f"`{first_line}`\n```{HANG_LIST[num_mistakes]}```\n{NEWLINE.join(player_msg_list)}"
+                        )
+                        await hg_msg.edit(embed=txt_embed)
+                    elif curr_guess not in incorrect_guesses:    # curr_guess not in not_guessed and not in word (elif) and not in incorrect guesses => curr_guess is incorrect and new
+                        num_mistakes += 1
+                        incorrect_guesses.add(curr_guess)
+                        last_line = f"incorrect guesses: {str(sorted(incorrect_guesses))[1:-1]}"
+                        player_msg_list.append(
+                            f"{curr_msg.author} guessed '{curr_guess}' wrong!")
+                        if len(player_msg_list) > 3:
+                            player_msg_list = player_msg_list[-3:]
+                        timeout_dict[curr_msg.author] = time()
+                        txt_embed.set_field_at(
+                            0,
+                            name=f"hangman (category: {cat_name})",
+                            value=
+                            f"`{first_line}`\n```{HANG_LIST[num_mistakes]}```\n{NEWLINE.join(player_msg_list)}"
+                        )
+                        txt_embed.set_footer(text=last_line)
+                        await hg_msg.edit(embed=txt_embed)
+                        if num_mistakes == 6:
+                            await ctx.send(
+                                f"sorry everyone, {curr_msg.author} used your last chance, the right answer was `{word}`"
+                            )
+                    else:    # curr_guess not in not_guessed and not in word and in incorrect_guesses (else) => curr_guess is incorrect but already made
+                        timeout_dict[curr_msg.author] = time()
+                        player_msg_list.append(
+                            f"{curr_msg.author}, '{curr_guess}' was already guessed"
+                        )
+                        if len(player_msg_list) > 3:
+                            player_msg_list = player_msg_list[-3:]
```

Comment:
`> 3` thing again
also i just realized this if statement isnt needed at all, since e.g. `["a", "b"][-3:]` is just `["a", "b"]`

Reply:
good point, added